From: Sven Roederer <devel-sven@geroedel.de>
Date: Tue, 9 Nov 2021 21:39:14 +0100
Subject: ath79: tiny: use 64 KiB SPI NOR erase sectors

This patch removes CONFIG_MTD_SPI_NOR_USE_4K_SECTORS from the default
symbols for the ath79/tiny target.

Based on 68d91f08edd0efd47841a2e53d3437d570a61612 / 23cd9fafa89a8058a88763cc0526a1e85a9f7ebb this
also removes 4K_SECTORS for in order to work around loosing config over sysupgrade.

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index 242719856d4112c029b37b4a9379472a47b124c4..79e8048d052882efb0d08653779750e1f13fb7d3 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -1,5 +1,4 @@
 CONFIG_LEDS_RESET=y
-CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
 CONFIG_NET_DEVLINK=y
 CONFIG_NET_DSA=y
 CONFIG_NET_DSA_MV88E6060=y
